static/js/services/http-wrapper.js

```diff
@@ -53,17 +58,20 @@ var _ = require('underscore');
        */
       return {
         get: function(url, options) {
-          return wrap([ url, options ], $http.get);
+          var updateTarget = options ? options.updateTarget : null;
+          return wrap([ url, options ], $http.get, updateTarget);
```

Comment:
The options is already being passed in, so you could use `args[args.length - 1].updateTarget` at line 46 and not pass it in again separately.